server/service/frontend.go

```diff
@@ -68,6 +70,71 @@ func ServeFrontend(urlPrefix string, sandbox bool, logger log.Logger) http.Handl
 	})
 }
 
+func ServeEndUserEnroll(urlPrefix string, logger log.Logger) http.Handler {
```

Comment:
Might be good to add `OTA` to this function name since the concept of "enrollment" is so overloaded.